provider/virtio_fs_utils.py

```diff
@@ -78,6 +82,8 @@ def basic_io_test(test, params, session):
                                        io_timeout).strip().split()[0]
         error_context.context("md5 of the guest file: %s" % md5_guest,
                               LOG_JOB.info)
+        if not os.path.isfile(host_data):
+            test.fail("The file %s does NOT exist!" % host_data)
```

Comment:
what about moving these 2 loc right after line72, because when we dd a file inside guest, this file should be accessible on the host, if that is not the case we dont need to compare the checksums